examples/CivisML_parallel_training.ipynb

```diff
@@ -149,11 +150,11 @@
     "COL_NAMES = ['sample_id', 'clump_thickness', 'uniformity_cell_size', 'uniformity_cell_shape',\n",
     "            'marginal_adhesion', 'epithelial_cell_size', 'bare_nuclei', 'bland_chromatin',\n",
     "            'normal_nucleoli', 'mitoses', 'is_cancer']\n",
-    "df = pd.read_csv(\"breast_cancer_data.csv\", names=COL_NAMES, na_values='?')\n",
+    "df = pd.read_csv(\"breast_cancer_data.csv\", names=COL_NAMES, na_values=np.nan)\n",
```

Comment:
Yep. Sorry, forgot it's for the CSV encoding rather than how you want pandas to encode it. Fixed now.